Intersect (Core)/GameObjects/Events/Commands/EventCommands.cs

```diff
@@ -1010,4 +1010,14 @@ public partial class CastSpellOn : EventCommand
 
         public bool GuildMembers { get; set; }
     }
+
+    public partial class FadeInCommand : EventCommand 
+    {
+        public override EventCommandType Type { get; } = EventCommandType.FadeIn;
+    }
+
+    public partial class FadeOutCommand : EventCommand
+    {
+        public override EventCommandType Type { get; } = EventCommandType.FadeOut;
+    }
```

Comment:
Can we just have a single Fade command with an enum (enum is `None, Out, In`)?

The intent is that eventually I would also want to add fade functions so the dev can control how much, how fast, etc it fades in/out. Easier to do on 1 class.

Reply:
Done, added those functions